### PR TITLE
feat(pr): pr测试

### PR DESCRIPTION
**Pull request by shannai0128**
this is a pr test

### CODE REVIEW

File: app/pr_app.go [hunk 1]
```diff
@@ -0,0 +1,7 @@
+package app
+
+import "fmt"
+
+func pr_app()  {
```
Comment (c479096292) [inline app/pr_app.go:5]: are three have some error?

Comment (shannai0128) [reply app/pr_app.go:5]: got it

**Comment by shannai0128** [comment]
i think it's ok!

**Comment by c479096292** [comment]
approve

**Comment by c479096292** [comment]
> i think it's ok!

enm.......

**Pull request merged**